duckduckgo/iOS
# Network Protection design review

Task/Issue URL: https://app.asana.com/0/72649045549333/1204630829332227/f

**Description**:

This adds a bunch of final tweaks to the UI + other changes to the project that allow a build to be shipped through the Alpha channel so it can be design reviewed.

**Steps to test this PR**:
1. Go through the NetP flow described in [the smoke testing steps](https://app.asana.com/0/0/1205201898139906/f)
2. Check everything matches [the designs](https://www.figma.com/file/zuCBX2UpJpS8CgtcmpU1hf/Network-Protection-for-iOS?type=design&node-id=218-19356&mode=design&t=Ywa7UsuRyJRbK9Dq-0)

**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


—
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
